inputs.tf

```diff
@@ -68,6 +68,12 @@ variable "minimum_node_count" {
   description = "Minimum nodes for the node pool. This is the total nodes so for regional deployments it is the total nodes across all zones."
 }
 
+variable "nat_ip_addresses" {
+  type        = list(string)
+  description = "List of IP addresses to use for NAT"
```

Comment:
What is the input value here?  The naming and suggestion suggests an actual IP addresses (1.2.3.4), the docs say that a `.self_link` to an `google_compute_address` resource of the correct purpose (`EXTERNAL`) is given.

Reply:
you're right, I've changed it to self_link now